### DISCUSSION TITLE
WSL<->Windows filesystem 的权限

### DISCUSSION THREAD

**Discussion by heroxbd** [physics-data/faq#54] [Q&A] [answered]
我发现从 WSL 发起保存在  `/mnt/d/` 的 LaTeX 文件，经过 Windows 系统上的 TexLive 操作后，权限从 WSL 里看都变成了 `775`。这有什么办法避免吗？

**Comment by jiegec** [comment]
在 Windows 上操作 Windows 文件，在 WSL 上操作 WSL 文件。

**Comment by Harry-Chen** [comment, marked as answer]
所有 drvfs 的文件，默认都显示为 755，实际是否可以访问还要看你的用户的 NTFS 权限。

如果需要在 drvfs 更改 Linux 权限，可以查看以下文章：

<https://devblogs.microsoft.com/commandline/chmod-chown-wsl-improvements/>
